types/authorization.yaml

```diff
@@ -32,6 +32,7 @@ types:
       - Processing
       - Accepted
       - Rejected
+      - TokenTimedOut
```

Comment:
Maybe add a comment, that it happens, when the validation was successful, but no charging session was initiated?

Reply:
Done :)